neqo-transport/src/connection.rs

```diff
@@ -54,7 +54,12 @@ const CID_LENGTH: usize = 8;
 
 pub const LOCAL_STREAM_LIMIT_BIDI: u64 = 16;
 pub const LOCAL_STREAM_LIMIT_UNI: u64 = 16;
+
+#[cfg(not(test))]
 const LOCAL_MAX_DATA: u64 = 0x3FFF_FFFF_FFFF_FFFE; // 2^62-1
+#[cfg(test)]
+const LOCAL_MAX_DATA: u64 = 0x3FFF; // 16,383
```

Comment:
I'd be happier with a setter than using `#[cfg(test)]`.  Defaulting to VARINT_MAX is fine.  The tests can then explicitly set the limit lower.

Reply:
opened #191, for future improvement.